README.md

```diff
@@ -2,30 +2,177 @@
+literally every day.
+
+It is ***vital*** that you be skilled with arrays.
+
+Let's establish an example for the rest of this lesson: the Shark Family as
+ordered by age, youngest to oldest.
+
+1. "Baby Shark"
+2. "Mama Shark"
+3. "Papa Shark"
+4. "Grandma Shark"
+5. "Grandpa Shark"
+
+## Recognize Array Vocabulary Word: Element
+
+The individual members of an array are called _elements_. In our example, "Baby
+Shark" is an _element_. "Grandma Shark" is also an element.
+
+## Recognize That Arrays Are Ordered By Index
+
+Within an array, ***elements are ordered***. The elements each have an _index_
+which stores their position in the array.  "Baby Shark" is the first _element_.
+"Grandpa Shark" is the last _element_.
+
+Some programming languages start counting the first element at `1`.  Most start
+counting the first element at `0`.
+
+We will show how to access elements in an array a little later in this lesson.
+
+## Recognize `[]` as Used for Presenting Arrays
+
+In Ruby we display an `Array`'s contents by using square-brackets(`[]`).
+
+As an example, we might want to show an `Array` of the Shark Family as ordered
+by age from youngest to oldest:
+
+> **TYPOGRAPHICAL NOTE** When you see a bit of text `Like This` it means we're
+> talking about a Ruby class. An "array" is a concept, but _in Ruby_ that concept
+> is used by working with the `Array` class.
+
+```ruby
+[ "Baby Shark", "Mama Shark", "Papa Shark", "Grandma Shark", "Grandpa Shark" ]
+```
+
+## Recognize That Array Elements Can Be Accessed Through Bracket-notation
 
-### Recognize Array As a Core Tool in Ruby
+Given an `Array` like:
 
-* we'll want to store collections of characters, rows of data in a file, people
-  who liked your post on social media
-* over and over again, this is a core behavior of "doing programming"
-  transforming the specific elements of an Array somehow
+```ruby
+shark_family = [ "Baby Shark", "Mama Shark", "Papa Shark", "Grandma Shark", "Grandpa Shark" ]
+```
 
-### Demonstrate Array Creation in Ruby
+We can get a single element by use of _bracket notation_.
 
-* Array.new
-* `[]`
-* Mention here that the elements are "in order" based on the position
-  0..Infinity
+```ruby
+shark_family[0] #=> "Baby Shark"
+shark_family[2] #=> "Papa Shark"
+shark_family[-1] #=> "Grandpa Shark"
+```
+
+> **TYPOGRAPHICAL NOTE** When you see `#=>` it means "this is what the
+> programming language returns. It's a common syntax for helping programmers
+> who are reading documentation understand how something works.
+
+## Recognize Array Creation
+
+Ruby has two primary paths for creating `Array`s from scratch:
+
+* Array literal syntax
+* `Array.new`
+
+### Array Literal Syntax
+
+Because showing arrays with brackets is so handy, Ruby added the ability to
+_create_ arrays by simply typing this naturally-readable display content.
+"Literal" here means "readable" like the word "literature" or "literacy." It's
+creating an `Array` by "reading it in."
```

Comment:
I'm not sure of 'reading it in' is clear enough of an explanation. Maybe a narrative would be clearer like - 'you can imagine saying in your head, "I want an array that _literally_ looks like this when displayed."'